app/styles/ui/history/_expandable-commit-summary.scss

```diff
@@ -139,20 +139,17 @@
       font-size: var(--font-size-sm);
       flex-shrink: 0;
 
-      .avatar,
-      .octicon {
-        display: inline-block;
-        margin-right: var(--spacing-third);
-        vertical-align: bottom; // For some reason, `bottom` places the text in the middle
-      }
```

Comment:
Turns out due to css hierarchy this was not being applied in production and when applied messes up the avatar stacking.